robottelo/utils/vault.py

```diff
@@ -25,7 +25,7 @@ def __init__(self, env_file='.env'):
     def setup(self):
         if self.env_path.exists():
             self.envdata = self.env_path.read_text()
-            is_enabled = re.findall('\nVAULT_ENABLED_FOR_DYNACONF=(.*)', self.envdata)
+            is_enabled = re.findall('VAULT_ENABLED_FOR_DYNACONF=(.*)', self.envdata)
```

Comment:
I had to play with this a bit...
I suggest using `^[\s]*VAULT_ENABLED_FOR_DYNACONF=(.*)`

Reply:
`^` works only when it is on the first line since
`^` means start of the string not start of the line

Reply:
but using flag `re.MULTILINE` it can be changed to expected behavior -  `^` means start of the line